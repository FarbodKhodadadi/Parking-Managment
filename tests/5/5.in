request_spot car55
assign_spot 11
request_spot car1
request_spot car5
assign_spot 33
checkout 11
pass_time 2
request_spot car5
checkout 33
