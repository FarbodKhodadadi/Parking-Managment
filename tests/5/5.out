11: normal 100 20
44: covered 150 50
Spot 11 is occupied now.
33: covered 250 70
44: covered 150 50
Spot 33 is occupied now.
Spot 11 is free now.
Total cost: 100
11: normal 100 20
44: covered 150 50
Spot 33 is free now.
Total cost: 390
